spec/dummy/config/initializers/schema_validations.rb

```diff
@@ -0,0 +1,2 @@
+require "#{Gem::Specification.find_by_name('rad_common').gem_dir}/lib/core_extensions/active_record/base/schema_validations"
+ActiveRecord::Base.prepend CoreExtenions::ActiveRecord::Base::SchemaValidations
```

Comment:
needed in all apps to patch active record

Reply:
We now have app/initializers/rad_common.rb for things that need to be in all apps, can you see if this works if the code is moved into there?

Reply:
moved, seems to work just the same